index.js

```diff
@@ -46,25 +73,13 @@ panel.port.on('addon-message', opts => {
     panel.hide();
   } else if (title === 'close') {
     panel.port.emit('set-video', {domain: '', src: ''});
+    // If the panel has been minimized, reset it to full height.
+    adjustHeight(180);
```

Comment:
Should probably make `180` a constant since we're using it 3x in this page.
And on that note, we should probably do a constant for width of 320 since we use it 2x.